pipe_anchorages/transforms/create_port_visits.py

```diff
@@ -104,7 +104,7 @@ def create_port_visits(self, tagged_events):
         grouping_id, events = tagged_events
         if not len(events):
             return
-        id_ = events[0].ssvid, events[0].vessel_id, events[0].seg_id
+        id_ = events[0].ssvid, events[0].vessel_id
```

Comment:
Is related with `Improve labelling of events inside port visits, often which had incorrect seg_id`?